model/pom.xml

```diff
@@ -75,6 +85,23 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.hibernate</groupId>
+      <artifactId>hibernate-validator</artifactId>
+      <scope>test</scope>
+    </dependency>
+
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <ignoredUnusedDeclaredDependencies>io.fabric8.funktion:funktion-model</ignoredUnusedDeclaredDependencies>
```

Comment:
perhaps can be removed, marks that funktion-model can be declared as a dependency even if it is not used directly